contributing.md

```diff
@@ -3,15 +3,15 @@
 Please ensure your pull request adheres to the following guidelines:
 
 - Search previous suggestions before making a new one, as yours may be a duplicate.
-- Make sure your list is useful before submitting. That implies it having enough content and every item a good succinct description.
+- Make sure your list is useful before submitting. That implies it has enough content and every item a good concise description.
 - A link back to this list from yours, so users can discover more lists, would be appreciated.
 - Make an individual pull request for each suggestion.
 - Titles should be [capitalized](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html).
-- Use the following format: `[List Name](link)`
-- Link additions should be added to the bottom of the relevant category.
+- Use the following format: '[List Name](link)'
```

Comment:
I think the goal here is explaining how markdown links work, but if we use single quotes the link is not escaped.
```suggestion
- Use the following format: `[List Name](link)`
```